crates/gas/src/call_graph/node.rs

```diff
@@ -0,0 +1,246 @@
+use std::cell::{Ref, RefCell, RefMut};
+use std::collections::HashSet;
+use std::fmt::{Debug, Display};
+use std::hash::{Hash, Hasher};
+use std::rc::Rc;
+
+use super::Value;
+
+/// This struct implement a Node for the `CallGraph`.
+///
+/// Each node has:
+///
+/// * A alue associated with it. This value is assumed to be unique across.
```

Comment:
You seem to have a typo here, and the connection between the 2 sentences isn't clear enough. Also, you described only the value property, while it looks as if you intended to described all properties.